docs/source/examples/simple-landscape-braket.md

```diff
@@ -0,0 +1,203 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Using ZNE to compute the energy landscape of a variational circuit with Braket
+
+This tutorial shows an example in which the energy landscape for a two-qubit variational circuit is explored with and without error mitigation, using Braket as our frontend.
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+import numpy as np
+
+from braket.devices import LocalSimulator
+from braket.circuits import Circuit, Noise, Observable
+
+from mitiq.zne import mitigate_executor
```

Comment:
Since Mitiq has many different mitigation techniques, I worry importing `mitigate_executor` could obscure which technique we are using down the line. What if we import `zne`, then use `zne.mitigate_executor` when we need this function?
```suggestion
from mitiq import zne
```